pkgs/by-name/ni/ninja/package.nix

```
@@ -11,18 +11,35 @@
 , buildPackages
 , buildDocs ? true
 , nix-update-script
+, ninjaRelease ? "1.12"
 }:
 
+let
+  releases = {
+    # TODO: Remove Ninja 1.11 as soon as possible.
+    "1.11" = fetchFromGitHub {
+      owner = "ninja-build";
+      repo = "ninja";
+      rev = "v1.11.1";
+      hash = "sha256-LvV/Fi2ARXBkfyA1paCRmLUwCh/rTyz+tGMg2/qEepI=";
+    };
+
+    "1.12" = fetchFromGitHub {
```

```suggestion
    "latest" = fetchFromGitHub {
```

this one should be the latest version, we're not trying to pin it to any specific version.

---

Fair enough; pushed. Hopefully this is gone by the time 1.13 comes around either way.